### DISCUSSION TITLE
Is there an equivalent of conda-pack?

### DISCUSSION THREAD

**Discussion by janton01** [prefix-dev/pixi#1369] [Q&A] [answered]
Is there an equivalent of the conda-pack command (https://conda.github.io/conda-pack/) that helps deploy environments on systems that may not have conda, or now pixi, already installed? thanks!

**Comment by ruben-arts** [comment, marked as answer]
We don't have that yet, but for the time being the lock file in pixi allows you to have a guaranteed copy of a working environment. So if you can make `pixi` available on the production system then you would just need to clone the project and `pixi install` to get it to the same stage. (assuming you don't use a `post-install` task which might not be locked into the `pixi.lock`).

**Comment by lucascolley** [comment]
https://pixi.sh/latest/deployment/pixi_pack/
